src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -60,23 +60,27 @@
 
 (def ^:private ALL_USERS "All users")
 
+(defn- make-list [input]
+  ;;input may or maynot be a list, make it a list
+  (flatten (conj '() input)))
```

Comment:
`flatten` is a little dangerous, as it flattens recursively.  `(flatten [1 [2 [3] 4] 5])` -> `[1 2 3 4 5]`  Also, it looks like we don't need lists specifically, just any sequential type.  I would suggest:

```clojure
(defn- ensure-sequence [input]
  (if (sequential? input)
    input
    [input]))
```

And then replace `'()`s with `[]`s.